x/distribution/keeper/key.go

```diff
@@ -7,49 +7,6 @@ import (
 	"github.com/cosmos/cosmos-sdk/x/distribution/types"
 )
 
-const (
-	// default paramspace for params keeper
-	DefaultParamspace = types.ModuleName
-)
-
-// Keys for distribution store
-// Items are stored with the following key: values
-//
-// - 0x00<proposalID_Bytes>: FeePol
-//
-// - 0x01: sdk.ConsAddress
-//
-// - 0x02<valAddr_Bytes>: ValidatorOutstandingRewards
-//
-// - 0x03<accAddr_Bytes>: sdk.AccAddress
-//
-// - 0x04<valAddr_Bytes><accAddr_Bytes>: DelegatorStartingInfo
-//
-// - 0x05<valAddr_Bytes><period_Bytes>: ValidatorHistoricalRewards
-//
-// - 0x06<valAddr_Bytes>: ValidatorCurrentRewards
-//
-// - 0x07<valAddr_Bytes>: ValidatorCurrentRewards
-//
-// - 0x08<valAddr_Bytes><height>: ValidatorSlashEvent
-var (
-	FeePoolKey                        = []byte{0x00} // key for global distribution state
-	ProposerKey                       = []byte{0x01} // key for the proposer operator address
-	ValidatorOutstandingRewardsPrefix = []byte{0x02} // key for outstanding rewards
-
-	DelegatorWithdrawAddrPrefix          = []byte{0x03} // key for delegator withdraw address
-	DelegatorStartingInfoPrefix          = []byte{0x04} // key for delegator starting info
-	ValidatorHistoricalRewardsPrefix     = []byte{0x05} // key for historical validators rewards / stake
-	ValidatorCurrentRewardsPrefix        = []byte{0x06} // key for current validator rewards
-	ValidatorAccumulatedCommissionPrefix = []byte{0x07} // key for accumulated validator commission
-	ValidatorSlashEventPrefix            = []byte{0x08} // key for validator slash fraction
-
-	ParamStoreKeyCommunityTax        = []byte("communitytax")
-	ParamStoreKeyBaseProposerReward  = []byte("baseproposerreward")
-	ParamStoreKeyBonusProposerReward = []byte("bonusproposerreward")
-	ParamStoreKeyWithdrawAddrEnabled = []byte("withdrawaddrenabled")
-)
-
 // gets an address from a validator's outstanding rewards key
 func GetValidatorOutstandingRewardsAddress(key []byte) (valAddr sdk.ValAddress) {
```

Comment:
shouldn't all these functions be moved to `/types` too? I don't see why they should belong to the keeper